deployments/fargate/README.md

```diff
@@ -205,6 +205,9 @@ allows you to collect telemetry data by deploying the Collector in a task that i
 from tasks containing monitored applications. This is in contrast to the sidecar deployment
 where the Collector container, and the monitored application containers are in the same task.
 Do not configure the ECS
-[resourcedetection](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/processor/resourcedetectionprocessor#resource-detection-processor) 
+[resourcedetection](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/processor/resourcedetectionprocessor#resource-detection-processor)
 processor for the standalone task since it would detect resources in the standalone Collector
 task itself as opposed to resources in the tasks containing the monitored applications.
+
+### AWS Graviton2
+We support [Fargate Graviton2](https://aws.amazon.com/ec2/graviton/) with the arm64 collector docker image with the default Fargate configuration.
```

Comment:
```suggestion
We support [Fargate Graviton2](https://aws.amazon.com/ec2/graviton/) with the default Fargate configuration. Splunk OpenTelemetry Collector docker image can run on both amd64 and arm64 architectures.
```

Reply:
Sounds good, the initial thought was to focus to arm support since it is relevant to Graviton - this works as well since it gives a general overview of support. Thanks!